### ISSUE TITLE
You can trap people inside pizza boxes

### ISSUE THREAD

**Issue by moonheart08** [space-wizards/space-station-14#5255]
## Description
and no, they cannot escape either.
![image](https://user-images.githubusercontent.com/7806367/141158261-96489c80-7385-4036-98f4-69c00445eb8d.png)
![image](https://user-images.githubusercontent.com/7806367/141158409-8cd49bee-3445-4e13-9512-b333339ad9e8.png)

**Comment by moonheart08** [comment]
Did a quick review, the Pizza box is the only object with this issue.

**Comment by 20kdc** [comment]
So I just determined what the problem is:
The problem is that automatic open-on-move is globally broken, but pizza boxes are the only thing that requires you use an explicit open verb.

**Comment by moonheart08** [comment]
I think the fact you can put people inside at all is also part of the issue, but makes sense

**Comment by moonheart08** [comment]
it's an unusual way of "itemizing" a normally non-item object.

**Comment by 20kdc** [comment]
Well it is interesting how pizza boxes effectively offer infinite storage because they bypass the anti-recursion measures, but...

**Comment by Ygg01** [comment]
> Well it is interesting how pizza boxes effectively offer infinite storage because they bypass the anti-recursion measures, but...

Pizza box sponsored by Dimensional Bag Inc. Towards a singularity-free infinite storage.

**Comment by 20kdc** [comment]
So the geniuses at Bugs, Inc. have come up with a new scheme
turns out when an EntityStorage releases an item - *or person*, it can release it into the container one level above... without any verification that said container ought to be able to accept said item

TLDR: you can trap people in your backpack via pizza boxes

at this point I have half a mind to suggest adding a failsafe - if attempting to remove an item from a backpack that won't fit in your hand, just drop it
because I suspect this is going to be the kind of issue where this specific situation is fixed and another bunch remain lurking
(I'm reasonably sure bluespace body bags are intended to be capable of at least similar shenanigans, so best to deal with this early)

**Comment by 20kdc** [comment]
Oh, right, just to clarify people trapped in backpacks cannot be removed, willingly or otherwise
